aws/resource_aws_ec2_transit_gateway_peering_attachment.go

```diff
@@ -0,0 +1,161 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
+)
+
+func resourceAwsEc2TransitGatewayPeeringAttachment() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2TransitGatewayPeeringAttachmentCreate,
+		Read:   resourceAwsEc2TransitGatewayPeeringAttachmentRead,
+		Update: resourceAwsEc2TransitGatewayPeeringAttachmentUpdate,
+		Delete: resourceAwsEc2TransitGatewayPeeringAttachmentDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"peer_account_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
```

Comment:
You should be able to add `ValidateFunc: validateAwsAccountId,` here.

Reply:
Thanks @ewbankkit, had an extremely busy month, fortunately someone has PR'd these to my branch now.